src/pages/components/tabs/usage.mdx

```diff
@@ -451,6 +451,71 @@ secondary labels with line tabs or auto-width contained tabs.
 </Column>
 </Row>
 
+### Dismissible tabs
+
+Dismissible tabs allow users to close tabs providing a focused and relevant
+experience. Users can add or remove tabs as needed, accommodating future content
+additions or modifications without drastically changing the overall layout or
+structure.
+
+<Row>
+<Column colLg={8}>
+
+![Dismissible tabs](images/tabs-usage-15.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example of dismissible line tabs (top) and dismissible contained tabs (bottom)
+</Caption>
+
+#### When to use
+
+- Use dismissible tabs to offer flexibility and scalability in complex
+  interfaces that require users to create multiple sections or modules.
+- Use dismissible tabs content created or curated by the user.
+- Use to focus a specific data set or search results.
+
+#### When not to use
+
+- Do not use when tabs contain frequently used or critical information
+- Do not use as navigation
+
+#### Considerations
```

Comment:
I think for this section it woud be more consumable if you just went ahead and made these each an H4. I think its fine if they don't have an h3 to group them. 

<img width="1007" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/11670886/5f31b46d-ab80-4233-a2ee-cb73ff8005f3">